apps/admin/frontend/src/api.ts

```diff
@@ -256,42 +260,37 @@ export const getWriteInCandidates = {
   },
 } as const;
 
-interface GetWriteInDetailViewInput {
-  castVoteRecordId: Id;
-  contestId: Id;
-  writeInId: Id;
-}
-export const getWriteInDetailView = {
-  queryKey(input?: GetWriteInDetailViewInput): QueryKey {
+type GetWriteInImageViewInput = QueryInput<'getWriteInImageView'>;
+export const getWriteInImageView = {
+  queryKey(input?: GetWriteInImageViewInput): QueryKey {
     return input
-      ? [
-          'getWriteInDetailView',
-          input.castVoteRecordId,
-          input.contestId,
-          input.writeInId,
-        ]
-      : ['getWriteInDetailView'];
-  },
-  invalidateRelatedWriteInDetailViewQueries(
-    queryClient: QueryClient,
-    { castVoteRecordId, contestId, writeInId }: GetWriteInDetailViewInput
-  ) {
-    return queryClient.invalidateQueries({
-      predicate(query) {
-        return (
-          query.queryKey[0] === 'getWriteInDetailView' &&
-          query.queryKey[1] === castVoteRecordId &&
-          query.queryKey[2] === contestId &&
-          query.queryKey[3] !== writeInId
-        );
-      },
-    });
```

Comment:
Previously there was some extra logic to strategically invalidate the `DetailView` cache, because adjudicating a write-in can change the options for a write-in in the same ballot, same contest, but not other ballots or contests. But I think there was a lot of possible footguns, and the split between `Image` and `AdjudicationContext` allows us to prefetch and not invalidate the big data requests - the images - and always invalidate the `AdjudicationContext`.